src/plugins/intel_gpu/src/graph/impls/ocl_v2/group_normalization_fsv16.cpp

```diff
@@ -42,11 +41,19 @@ class GroupNormalizationGeneratorBase : public KernelGenerator {
 public:
     explicit GroupNormalizationGeneratorBase(std::string_view name, std::string_view suffix) : KernelGenerator(name, suffix) {}
     [[nodiscard]] JitConstants get_jit_constants(const RuntimeParams& params) const override {
+        auto get_max_simd_size = [](const RuntimeParams& params) {
+            size_t max_simd_size = fsv;
+            for (auto& simd_size : params.get_device_info().supported_simd_sizes) {
+                max_simd_size = std::max(max_simd_size, simd_size);
```

Comment:
Is this strategy beneficial for all platforms?

Reply:
Hi @sshlyapn, it looks like that it's beneficial for most platforms.
I measured execution time of group_normalization layers in the unet of SD 1.5, and summarized improvement ratio as below:

  | dynamic | static
-- | -- | --
MTL | 2% | 5%
DG2 | 11% | 11%
LNL | 38% | 74%
BMG | 39% | 86%
PTL | 38% | 54%